### PR TITLE
ENG-368 create the tutorial

### PR DESCRIPTION
**Pull request by Kerruba**

### CODE REVIEW

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Use Blueprint generated plugin and microfronteds without a bundle
+
+## Purpose
+
+In this tutorial you will learn how to use the plugin and the microfronteds generated using the Entando Blueprint
+in a running cluster without the need to include the components in a bundle 
+
+## Requirements
+
+-   A microservice built with the Entando Blueprint
```
Comment (joewhite101) [inline vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:10]: Blueprint --> Entando Component Generator (see glossary)

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+-   Node and NPM are installed on your machine
```
Comment (joewhite101) [inline vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:12]: Versions?

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+-   Docker installed on your machine and you are able to upload images
```
Comment (joewhite101) [inline vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:14]: docker-hub ... or an image repository of your choice?

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+## Steps
+
+## 1. Generate an entity using the Blueprint
+
+In order to include auto-generated micro frontends to your bundle you
+will need to use the `jhispter entity` generator. Let’s assume we want
+to generate a simple *Conference* entity.
+
+    jhipster entity conference
```
Comment (joewhite101) [inline vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:28]: Capital "C" at the start required? I'm not sure but have always capitalized entity names

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+- The EntandoPlugin generated with the blueprint has name `my-demo-plugin`
+- The EntandoApp where I want to expose the `my-demo-plugin` apis is named `my-entando-app` 
+- Both are deployed on the `entando` namespace. 
+- As a name for the link, I'll use `my-entando-app-to-my-demo-plugin-link`
+
+> **Warning**
+> Remember to change the fields to match your setup.
+
+```
+> cat <<EOF > /tmp/mylink.yaml
```
Comment (joewhite101) [inline vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:122]: Do we describe this CR structure elsewhere in the docs?

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md (hunk 4 above)
Comment (Kerruba) [reply vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:28]: Is not required, but I'll make it C

File: vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md (hunk 5 above)
Comment (Kerruba) [reply vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md:122]: Yeah, in the custom-resources document. I'll add a link to that document

**Pull request merged**